<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionMenu</class>
 <widget class="QMainWindow" name="connectionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>connectionMenu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>Database</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidget_2">
           <column>
            <property name="text">
             <string notr="true">Cloud Database</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
         <item>
          <widget class="QTableView" name="tableView_2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Connection(s)</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Connection features</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Driver</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Database Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Hostname</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLineEdit" name="nameLineEdit"/>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxDriver">
                  <item>
                   <property name="text">
                    <string>Select Driver</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QMYSQL</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QMYSQL3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QSQLITE</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QOCI8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QODBC3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QPSQL7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>QTDS7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>PHPMyAdmin</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Cloud9</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="dbNameLineEdit"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="usernameLineEdit"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="passwordLineEdit"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="hostLineEdit"/>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxPort"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="connectButton">
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closeButton">
                <property name="text">
                 <string>Close</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Table Ops</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="3" column="1">
                 <widget class="QToolButton" name="toolButton">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="res/res.qrc">
                    <normaloff>:/res/down.png</normaloff>:/res/down.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QToolButton" name="toolButton_4">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="res/res.qrc">
                    <normaloff>:/res/leftArrow.png</normaloff>:/res/leftArrow.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QToolButton" name="toolButton_2">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="res/res.qrc">
                    <normaloff>:/res/rightArrow.png</normaloff>:/res/rightArrow.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QToolButton" name="toolButton_3">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="res/res.qrc">
                    <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Db Ops</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Open Database</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sendImagesToTable">
                  <property name="text">
                   <string>Send to Table</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="progressBar">
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" alignment="Qt::AlignRight">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Use predefined connection</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Image(s)</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>718</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Zen On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Zen Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Connection status: not connected...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>321</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
